olive/workflows/run/run.py

```diff
@@ -231,28 +231,32 @@ def run_engine(package_config: OlivePackageConfig, run_config: RunConfig):
     # to remove engine level loop and pass the accelerator_specs to the engine directly.
     for accelerator_spec, (passes, pass_flows) in zip(acc_list, pass_list):
         engine.reset_passes()
-        if passes:
+        pass_flows_to_run = pass_flows or [[*passes.keys()]]
```

Comment:
can you combine this into a single set? There might be cases like `[["a", "b"], ["a", "b","c"]]` where we would be registering the same pass multiple times.

Reply:
I suppose this dedupe logic should be added to register method.